src/dags/asbest.py

```diff
@@ -88,8 +89,14 @@
 
     rename_col = PostgresOperator(task_id="rename_col", sql=SQL_RENAME_COL)
 
-load_dumps[0] >> rename_col >> rename_tables
-load_dumps[1] >> rename_tables
+    # Grant database permissions
+    grant_db_permissions = PostgresPermissionsOperator(
+        task_id="grants",
+        dag_name=dag_id
+    )
+
+load_dumps[0] >> rename_col >> rename_tables >> grant_db_permissions
+load_dumps[1] >> rename_tables >> grant_db_permissions
```

Comment:
Ik denk dat de grant_db_permissions beter aan het einde gezet kan worden, omdat de load van de tabel dan eerst plaatsvind. Het kan voorkomen dat de data in een tmp table gaat en daaran wordt gerenamed naar de echte tabel. In dat geval zou de permissies vooraf niet meer actief zijn.

Dus zoiets:
[ extract_shp >> convert_shp >> load_dump ] >> grant_db_permissions

Reply:
Het was al de laatste stap als ik het goed heb, maar de volgorde in de code is misschien een beetje verwarrend. 
Hij wordt namelijk pas uitgevoerd na de load_dump in de for loop (de rename volgt op load_dumps). 
Ik heb de regel met grant_db_permissions onder de for loop geschoven, zo komt hij ook na de rename, begrijp ik het zo goed?